.github/workflows/build_and_test.yml

```
@@ -24,27 +25,25 @@ jobs:
 
       - name: Run action-ros-ci
         id: action_ros_ci_step
-        uses: ros-tooling/action-ros-ci@v0.1
+        uses: ros-tooling/action-ros-ci@v0.2
         with:
           package-name: obstacle_stop_planner_refine
           target-ros2-distro: foxy
           vcs-repo-file-url: build_depends.repos
           colcon-mixin-name: coverage-gcc coverage-pytest
```

Since `action-ros-ci@v0.2`, we have to use `colcon-defaults` instead of `colcon-mixin-name` as follows:
https://github.com/ros-tooling/action-ros-ci#integrate-action-ros-ci-with-codecov
```suggestion
          colcon-defaults: |
            {
              "build": {
                "mixin": ["coverage-gcc", "coverage-pytest"]
              },
              "test": {
                "mixin": ["coverage-pytest"]
              }
            }
```

---

Thank you, I'll fix it.